CONTRIBUTING.md

```diff
@@ -13,57 +13,64 @@ Welcome to Pluto! We appreciate and welcome contributions on our project, your c
 # Setup the project locally
 
 1. Fork this repository into your Github account.
-2. Clone the forked repository to your local system using `git clone -b beta https://github.com/USERNAME/Pluto.git` where `USERNAME` is your actual github username.
-
-> [!WARNING]
-> Do not use normal `git clone https://github.com/USERNAME/Pluto.git` since it will directly clone the main branch of your forked repository and while the main branch is the stable version of the web app and beta branch is actually the development branch and that is where all the new changes will be committed.
-
+2. Clone the forked repository to your local system using `git clone https://github.com/USERNAME/Pluto-Web.git` where `USERNAME` is your actual github username.
 3. Navigate to `/server` directory which is the back-end of our web application, then run `npm install` (or) `yarn install` to install all dependencies.
-4. Also install `Nodemon` using `npm install -g nodemon`
-5. Create a file named `.env` in the `/server` directory itself, and paste the following into the file:
+4. Create a file named `.env` in the `/server` directory itself, and paste the following into the file:
 
 ```
 PORT=(The port number you want the backend to run in)
 MONGO_URI=(The uri of the MongoDB database)
 ```
 
 6. Replace the `PORT` environment variable with the port number you want the backend to run in, it is recommended to use the port number `4000` as it is the common port number used for back-end servers. For the `MONGO_URI` environment variable, you need to create an account on [MongoDB Website](https://www.mongodb.com/) and create a project, create a shared database _(the only free-tier database that MongoDB offers per project)_ and click connect button, click drivers and then choose the latest Node.js version and then copy the uri and paste it into the `MONGO_URI` environment variable.
-7. At last, you can run `npm run dev` to run the back-end server with nodemon or `npm run start` to run the back-end server with node.
+7. At last, you can run `npm run dev` to run the back-end server in nodemon.
+> [!IMPORTANT]
+> If you intend to run the back-end server in Node.js then you need to run `npm run start` but before that back-end server needs to be complied to JavaScript as it's TypeScript, so run `npm run build` and then proceed with `npm run start` to run in Node.js 
```

Comment:
If you intend to run the back-end server in Node.js, then you need to run `npm run start`. But before that, the back-end needs to be compiled to JavaScript as it's TypeScript. So run `npm run build` and then.....